docs_src/getting-started/Ch-GettingStartedUsersNexus.md

```diff
@@ -16,31 +16,31 @@ Reasons you might want to use container images from Nexus include:
 2.  You need the latest development container image (the work in progress)
 3.  You are working in a Windows or non-Linux environment and you are unable to build a container without some issues.
 
-A set of Docker Compose files have been created to allow you to get and use the latest EdgeX service images from Nexus.  Find these [Nexus "Nightly Build" Compose files](https://github.com/edgexfoundry/developer-scripts/tree/master/releases/nightly-build/compose-files) in GitHub.  The EdgeX development team provides these Docker Compose files.  As with the EdgeX release Compose files, you will find several different Docker Compose files that allow you to get the type of EdgeX instance setup based on: 
+A set of Docker Compose files have been created to allow you to get and use the latest EdgeX service images from Nexus.  Find these [Nexus "Nightly Build" Compose files](https://github.com/edgexfoundry/edgex-compose) in the `main` branch of the `edgex-compose` respository in GitHub.  The EdgeX development team provides these Docker Compose files.  As with the EdgeX release Compose files, you will find several different Docker Compose files that allow you to get the type of EdgeX instance setup based on: 
 
 - your hardware (x86 or ARM)
-- the database you want to use (Mongo or Redis)
 - your desire to have security services on or off
+- your desire to run with the EdgeX GUI included
 
 ![image](EdgeX_GettingStartedNexusCompose.png)
 
 !!! Warning
-    The "Nightly Build" images are provided as is and may not always function properly or with other EdgeX services.  Use with caution and typically only if you are a developer/contributor to EdgeX. These images represent the latest development work and may not have been thoroughly tested or integrated.
+    The "Nightly Build" images are provided as-is and may not always function properly or with other EdgeX services.  Use with caution and typically only if you are a developer/contributor to EdgeX. These images represent the latest development work and may not have been thoroughly tested or integrated.
 
 ## Using Nexus Images
-The operations to pull the images and run the Nexus Repository is the same as when using EdgeX images from Docker Hub (see [Getting Started with Docker](./Ch-GettingStartedUsers.md#run-edgex-foundry)).
+The operations to pull the images and run the Nexus Repository containers are the same as when using EdgeX images from Docker Hub (see [Getting Started with Docker](./Ch-GettingStartedUsers.md#run-edgex-foundry)).
```

Comment:
```suggestion
The operations to pull the images and run the Nexus Repository containers the same as when using EdgeX images from Docker Hub (see [Getting Started with Docker](./Ch-GettingStartedUsers.md#run-edgex-foundry)).
```

Reply:
?? think we still need `are` in the sentence.  If we take out all the extra words, the sentence would read:
"the operations to pull and run **are** the same as when ..."

Reply:
> ?? think we still need `are` in the sentence. If we take out all the extra words, the sentence would read:
> "the operations to pull and run **are** the same as when ..."

Ok, it is mouth full but now see that `are` refers to `operations`.